sdk-python/livescoreapi.py

```diff
@@ -7,17 +7,62 @@
 """
 
 import urllib.request
-
+import json
 
 class livescores:
+    """
+    A class used to get livescores. 
+    ...
+
+    Attributes
+    ----------
+    key : str
+        an alphanumeric string used to access data from the LiveScore API
+    secret : str
+        an alphanumeric string used to access data from the LiveScore API
     
-    def get_all(key,secret):
-        html_file = urllib.request.urlopen("http://livescore-api.com/api-client/scores/live.json?key=" + key + "&secret=" + secret)
-        json_data = html_file.read()
-        return json_data
+    Methods
+    -------
+    _get_html_file(link)
     
-
+    get_all()
+    
+    """
+    link_get_all = "http://livescore-api.com/api-client/scores/live.json?key={}&secret={}"
```

Comment:
We need the first part of the url to be separate `http://livescore-api.com/api-client` so we can reuse it for other API endpoints.